functions/src/plugins/size.ts

```diff
@@ -228,9 +225,9 @@ export class SizeTask extends Task {
         const reconstructArtifacts = (object: any, path: string) => {
           Object.keys(object).forEach(k => {
             if(typeof object[k] === 'object') {
-              reconstructArtifacts(object[k], path + '/' + k);
+              reconstructArtifacts(object[k], path + '/' + firebasePathDecode(k));
```

Comment:
how can you differentiate between existing "_" from the name, and the one that you added when you did the encode?
for example if the name was "bundle_2.br", it would be encoded into "bundle_2_br" and decoded into "bundle.2.br"
is it a problem?

Reply:
bugger.... You're right.
I don't know how you'd get around that, from what I can decipher there's isn't any special handling in the `payload-size.sh`